network/server_test.go

```diff
@@ -363,40 +365,51 @@ func WaitUntilPeerConnectsTo(ctx context.Context, srv *Server, id peer.ID) (bool
 // TestPeerReconnection checks whether the node is able to reconnect with bootnodes on losing all active connections
 func TestPeerReconnection(t *testing.T) {
 	conf := func(c *Config) {
-		c.MaxPeers = 2
+		c.MaxPeers = 3
 		c.NoDiscover = false
 	}
 	//Create bootnode
 	bootNode := CreateServer(t, conf)
-
+	bootNode2 := CreateServer(t, conf)
```

Comment:
Please consider not naming variables by prepending a number to them.

This practice is present for some reason in other tests as well, but we should look to move away from it

Reply:
Renamed them, Commit [link](https://github.com/0xPolygon/polygon-sdk/pull/260/commits/25709ccd3d899ace25567a64f7edf7488a934531)